packages/perseus-score/src/widgets/grapher/score-grapher.ts

```diff
@@ -16,7 +14,7 @@ function getCoefficientsByType(
         return undefined;
     }
     if (data.type === "exponential" || data.type === "logarithm") {
-        const grader = functionForType(data.type);
+        const grader = GrapherUtil.functionForType(data.type);
```

Comment:
`functionForType` is the reason for the size of the PR. Basically it was designed in a pattern similar to WidgetExports where a bunch of functionality and some React classes were thrown together and accessed via a central getter using the `type`.

This explains two things:

1. I had to move that getter and all associated code out of `perseus` because `perseus-score` can't import from `perseus`. `perseus` and `perseus-editor` still needed access to this logic, so I moved it to `perseus-core`.
2. `perseus-core` can't depend on React/JSX, so I had to split the data (similar to what we're doing with the scorers and WidgetExports); but I still needed access to the Movable component in Grapher, so I made the `movableToGraphieMovable` mapper.